### PR TITLE
Floatbar fixes and windows support

### PR DESCRIPTION
**Pull request by akallabeth**
* Allow more fine grained settings for X11 floatbar
* Integrate old pr #3419 and extend with new command line settings
* unify `cmdline.c` string to integer conversions and error checks

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
Tested the wfreerdp and xfreerdp parts. Both work. Windows has some (unrelated) problems though.

**Comment by bmiklautz** [comment]
@hardening any objections?

File: client/Windows/wf_client.c [hunk 1]
```diff
@@ -302,14 +305,48 @@ static void wf_add_system_menu(wfContext* wfc)
...
+	WCHAR prefix[] = L"FreeRDP:";
+
+	if (!settings)
+		return NULL;
+
+	name = settings->ServerHostname;
+
+	if (settings->WindowTitle)
+	{
+		WCHAR* windowTitle = NULL;
```
[the remark this answers is not in the document]
Comment (bmiklautz) [reply client/Windows/wf_client.c]: It's not necessary I think. @mfleisz good finding.

[the remark this answers is not in the document]
Comment (akallabeth) [reply client/Windows/wf_client.c]: @mfleisz good catch, fixed.

**Pull request merged**